### PR TITLE
mkosi: make sure networkd test mkosi works with f33

### PR DESCRIPTION
**Pull request by poettering**

### CODE REVIEW

File: test/mkosi.default.networkd-test [hunk 1]
```diff
@@ -57,6 +57,8 @@ BuildPackages=
         ninja-build
         pam-devel
         pcre2-devel
+        perl-IPC-SysV
+        perl-Time-HiRes
```
Comment (keszybz) [inline test/mkosi.default.networkd-test]: `perl(IPV::SysV)` and `perl(Time::HiRes)` would be more canonical (resilient to package splits and merges).

We should convert most of those `-devel` deps to virtual provides too.

**Comment by poettering** [comment]
updated as suggested. ptal.

**Comment by mrc0mmand** [comment]
I'll skip rescheduling the CentOS CI, since the changes are not tested there anyway, so please ignore the fails.

**Comment by keszybz** [comment]
Some CI passes, let's merge.

**Pull request merged**